lib/coverband/configuration.rb

```diff
@@ -60,7 +60,7 @@ def initialize
     end
 
     def reset
-      @root = Dir.pwd
+      @root = ENV["COVERBAND_ROOT"] || Dir.pwd
```

Comment:
Alternatively, if you did not want to introduce an additional env var, we could do:

```
@root = ENV["PWD"] || Dir.pwd
```

The difference is that `ENV["PWD"]` will return the name of the symoblic link, while `Dir.pwd` will return the path of the underlying directory.